thorlcr/master/thmastermain.cpp

```diff
@@ -441,9 +441,61 @@ bool checkClusterRelicateDAFS(IGroup &grp)
 
 static bool auditStartLogged = false;
 
+class CThorEndHandler : public CSimpleInterface, implements IThreaded
+{
+    CThreaded threaded;
+    unsigned timeout;
+    std::atomic<bool> started{false};
+    std::atomic<bool> stopped{false};
+    Semaphore sem;
+public:
+    CThorEndHandler() : threaded("CThorEndHandler"), timeout(30000)
```

Comment:
picky/trivial: better to initialize on member declaration.

Reply:
done